mm2src/coins/z_coin/z_coin_errors.rs

```diff
@@ -207,6 +207,7 @@ pub enum ZCoinBuildError {
     RpcClientInitErr(ZcoinClientInitError),
     ZCashParamsNotFound,
     ZDerivationPathNotSet,
+    ChecksumVerificationFailed,
```

Comment:
`ChecksumVerificationFailed` is too generic. I would call it `SaplingParamsInvalidChecksum` instead.

Reply:
done